content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,184 @@
+
+.. image:: rfq/RFQ-dashboard.png
+   :align: center
+   :alt: |RFQ| dashboard with orders and order statuses.
+
+In addition to various view options, the :guilabel:`Requests for Quotation` dashboard provides
+:guilabel:`Filters` and :guilabel:`Group By` options, accessible via the search bar drop-down menu.
+
+Create new |RFQ|
+----------------
+
+To create a new |RFQ|, click the :guilabel:`New` button on the top-left corner of the
+:guilabel:`Requests for Quotation` dashboard to reveal a new |PO| form.
+
+Start by assigning a :guilabel:`Vendor`.
+
+The :guilabel:`Vendor Reference` field points to the sales and delivery order numbers sent by the
+vendor. This comes in handy once products are received, and the |PO| needs to be matched to the
+delivery order.
+
+The :guilabel:`Blanket Order` field refers to long-term purchase agreements on recurring orders with
+set pricing. To view and configure blanket orders, head to :menuselection:`Purchase app --> Orders
+--> Purchase agreements`.
+
+The :guilabel:`Currency` can be changed, if purchasing products from a vendor in another country.
+
+Next, configure an :guilabel:`Order Deadline`, which is the date by which the vendor must confirm
+their agreement to supply the products.
+
+.. note::
+   After the :guilabel:`Order Deadline` is exceeded, the |RFQ| is marked as late, but the products
+   can still be ordered.
+
+:guilabel:`Expected Arrival` is automatically calculated based on the :guilabel:`Order Deadline`
+and vendor lead time. Tick the checkbox for :guilabel:`Ask confirmation` to ask for signage at
+delivery.
+
+With the :doc:`Storage Locations feature
+<../../inventory/warehouses_storage/inventory_management/use_locations>` activated,
+the :guilabel:`Deliver to` field appears, with options for the order shipment.
+
+Select the receiving warehouse address here, or select :guilabel:`Dropship` to indicate that this
+order is to be shipped directly to the end customer. When :guilabel:`Dropship` is selected, the
+:guilabel:`Dropship address` field is enabled. Contact names auto-populate here from the
+**Contacts** app.
+
+Products tab
+~~~~~~~~~~~~
+
+In the :guilabel:`Products` tab, add the products to be ordered. Click :guilabel:`Add a product`,
+and type in the product name, or select the item from the drop-down menu.
+
+To create a new product and add it, type the new product name in the :guilabel:`Product` column,
+select :guilabel:`Create [product name]` from the resulting drop-down menu, and manually add the
+unit price. Or, select :guilabel:`Create and edit...` to be taken to the product form for that new
+item.
+
+:guilabel:`Catalog` can also be selected to navigate to a product menu from the chosen vendor. From
+here, products can be added to the cart.
+
+.. note::
+  To make adjustments to products and prices, access the product form by clicking the
+  :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
+  the :guilabel:`Product` name.
```

Comment:
Text in an admonition needs to align with the first letter of the admonitions name, `n` in `.. note::` in this case :)
```suggestion
.. note::
   To make adjustments to products and prices, access the product form by clicking the
   :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
   the :guilabel:`Product` name.
```